opengrok-indexer/src/main/java/org/opengrok/indexer/authorization/LdapError.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ */
+
+package org.opengrok.indexer.authorization;
+
+/**
+ * Unchecked exception to be thrown when LDAP server pool is down.
+ */
+public class LdapError extends RuntimeException {
```

Comment:
why call it error when it's an exception?

Reply:
true, I wanted to rename but then forgot.